base_transaction_id/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2012 Yannick Vaucher, Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{'name': 'Base transaction id for financial institutes',
+ 'version': '11.0.1.0.0',
+ 'author': "Camptocamp,Odoo Community Association (OCA)",
```

Comment:
Separate authors w/ space, please

Reply:
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#modules

According to contribution guidelines this is correct 

``` python
author + ',Odoo Community Association (OCA)
```

Reply:
Odoo parses both with or without spaces, so no problem in any

Reply:
@pedrobaeza that just doesn't look good when it gets rendered, imho

Reply:
The parsing is the same as said. You can try.